dts/bindings/arm/arm,ethos-u.yaml

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2022 Arm Limited.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the License); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
General comment about this PR: I don't understand why it's mixing SPDX lines with verbatim licenses. This seems redundant and unnecessary. I think only the SPDX lines are needed. Am I missing something?

Reply:
Agree the SPDX line should be sufficient. I'll update the files.